rs_client.py

```diff
@@ -1019,6 +1019,7 @@ def _rs_struct(typeobj):
 
         _r('pub type %s%s = base::StructPtr<%s%s>;', typeobj.rs_type, lifetime1,
                 lifetime2, typeobj.ffi_type)
+        typeobj.struct_ptr = True
 
```

Comment:
I meant you could directly use `rs_is_pod` attribute instead of creating a new one.
However, if `rs_struct_ptr` helps readability, it for sure won't hurt.